contracts/Staking.sol

```diff
@@ -155,6 +159,15 @@ contract Staking is Ownable {
         }
     }
 
+    function setClaimFeeParams(
+        uint ownerFee,
+        uint stakeRegistryFee
+    ) external onlyOwner {
+        require(ownerFee.add(stakeRegistryFee) < 10000, "Invalid claim fees");
+        ownerFeeRate = ownerFee;
+        stakeRegistryFeeRate = stakeRegistryFee;
+    }
```

Comment:
Configurable fees are not in the scope of the task.
If they were, they'd need to rely on existing governance functions.